canonical/maas-ui
# chore: upgrade vanilla v4 MAASENG-2086

## Done

chore: upgrade vanilla v4
- update Ubuntu font assets
- update `.section-header__title` top padding to maintain correct alignement with machine list controls

https://vanillaframework.io/docs/migration-guide-to-v4#how-to-update

## QA

This will need thorough QA.
Try to visit all pages and look for anything unexpected, like items no longer being positioned correctly.

## Screenshots

### After
![image](https://github.com/canonical/maas-ui/assets/7452681/10169ec9-f1e3-46b8-af68-2d45b3b96507)

![image](https://github.com/canonical/maas-ui/assets/7452681/4fc50106-f81e-4862-8f42-3ec2e6bb70dd)
